NuRadioReco/detector/detector_base.py

```diff
@@ -931,9 +931,9 @@ def get_channel_group_id(self, station_id, channel_id):
         res = self.__get_channel(station_id, channel_id)
         if 'channel_group_id' not in res.keys():
```

Comment:
no need to use `.keys()` here if `res` is a dict.

Reply:
changed that in the new PR